### PR TITLE
Updating Script repo with latest ApiHub SDK

### PR DESCRIPTION
**Pull request by safihamid**
Conflicts:
	src/WebJobs.Script.Host/WebJobs.Script.Host.csproj
	src/WebJobs.Script.Host/packages.config
	src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj
	src/WebJobs.Script.WebHost/packages.config
	src/WebJobs.Script/GlobalSuppressions.cs
	src/WebJobs.Script/WebJobs.Script.csproj
	src/WebJobs.Script/packages.config
	test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj
	test/WebJobs.Script.Tests/packages.config

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@safihamid__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hamids). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: test/WebJobs.Script.Tests/packages.config [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
   <package id="Microsoft.Azure.WebJobs.Core" version="1.1.2-alpha-10289" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions" version="1.0.2-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.EasyTables" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="1.0.2-alpha-10268" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Extensions" version="1.0.2-alpha-10270" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-alpha-10270" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.0.0-alpha-10267" targetFramework="net46" />
```
Comment (davidebbo) [inline test/WebJobs.Script.Tests/packages.config]: Why are some packages like DocumentDB changing to older versions? That doesn't seem right.

**Comment by mathewc** [comment]
Curious what the "Conflicts" description is on the PRs you're sending. I'd rather just see a real description of the change.

**Comment by fabiocav** [comment]
This is also open against master. To accurately review the changes, the PR should be open against (and rebased on) dev.

**Pull request closed without merging**